packages/flutter_tools/test/src/schema.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/flutter_manifest.dart';
+
+/// Writes a schemaData, required for pubspec.yaml to be loaded.
```

Comment:
This should probably have a better doc comment. What is schemaData?

Reply:
I wasn't entirely sure, I just brought this comment with the code. However, looking more, it seems like the asset code validates pubspec.yamls using schemas. These methods are used in tests to write those schemas (and in most cases write an empty file that will pass any file).

I've tweaked the comments to be more descriptive.